### ISSUE TITLE
Option to auto-rename files to not mess up filename ordering

### ISSUE THREAD

**Issue by GoogleCodeExporter** [catgirlfighter/nekopaw#235]
```
For example, when downloading from exhentai, files will have original names 
like 1.jpg...9.jpg, 10.jpg, 100.jpg  
With all naming variables I've tried, the output file will be 
<galleryid>-<filename>.jpg, for example 12345-1.jpg....1235-10.jpg  
This causes issues in both windows ordering and viewer software ordering.  
I tried using the "%width:.6d% " example from the wiki, since the original 
filename is decimal, but it doesn't seem to work.  
For exhentai, the files have a "title" field with the number.  
So I tried the following:  
$rootdir$\pics\%album%\%title:%width:.4d%% (doesn't work at all)  
$rootdir$\pics\%album%\%title[:%width:.4d%]% (starts downloading but endlessly 
starts redownloading same file, so most likely not working)  
$rootdir$\pics\%album%\$fname[:%width:.4d%]$ (same as above)  
Maybe I'm misunderstanding width altogether, and width is the pictures X 
resolution if applicable, in which case it can't be used for renaming files 
anyways.  

In short, I want: 12345-1.jpg to be either 12345-001.jpg or just 001.jpg and 
12345-10.jpg to be 12345-010.jpg or 010.jpg.  
A program similar to yours, domdomsoft manga downloader, has a simple tickbox 
called "rename single digit to three digits" (also applies to 2 digits), 
basically this would be just what I need

```

Original issue reported on code.google.com by `confri...@gmail.com` on 21 Aug 2014 at 9:11
